hooks/command

```diff
@@ -30,14 +29,17 @@ buildkite-agent artifact download \
 
 echo "--- :junit: Processing the junits"
 
+# Move ruby code to current dir so we can mount it
+plugin_ruby_dir="$(cd "$( dirname "${BASH_SOURCE[0]}" )/../ruby" && pwd)"
+ruby_dir="$(pwd)/$(mktemp -d "junit-annotate-plugin-ruby-tmp.XXXXXXXXXX")"
+cp -R "${plugin_ruby_dir}" "${ruby_dir}"
```

Comment:
I guess you already know, but the original was trying to mount something in `/buildkite/plugins`, which is not [in the list of volumes we mount](https://github.com/harvesthq/ci-cd/blob/978ccda9673a22ede9edafb5a03cae15de4c7ebc/.kubernetes/manifests/buildkite-agent/deployment-buildkite-agent.yml.erb#L45-L55). My solution was to instead copy the ruby files to a folder inside the current build's path.

Do you like this?

Reply:
Sure! Although it looks like you only need the `annotate` script from the `ruby/` dir, right? If that's the case, I'd only copy that file and tweak the `command` script a bit:

```bash
annotate_src="${BASH_SOURCE%/*}/../ruby/bin/annotate"
annotate_path="${PWD}/.buildkite/bin/annotate-tests"
cp -a "${annotate_src}" "${annotate_path}"

docker \
  --log-level "error" \
  run \
  --rm \
  --volume "${annotate_path}:/usr/local/bin/anotate-tests" \
  --volume "${artifacts_dir}:/junits" \
  ruby:2.7-alpine ruby /usr/local/bin/anotate-tests /junits \
    > "$annotation_path"
```

I'm fine with mounting the `plugins` dir across Buildkite and Docker-in-Dockers too. Whatever's easier!